database/migrations/2020_12_03_151728_create_applicants_skills_table.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateApplicantsSkillsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('applicants_skills', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('applicant_id');
+            $table->foreign('applicant_id')
+                ->references('id')
+                ->on('applicants')
+                ->onUpdate('CASCADE')
+                ->onDelete('NO ACTION');
```

Comment:
I think this could be CASCADE. If the Applicant itself is deleted, applicant skills should be part of that.